update_repo.sh

```diff
@@ -55,15 +61,16 @@ update_repo()
 {
     OPTS=
     [ -r "$1/comps.xml" ] && OPTS="-g comps.xml"
-    $createrepo $OPTS --update -x 'fc*/*' -o $1 $1 >/dev/null
-    if [ -x "$mkmetalink" -a -r "$mirrors_list" ]; then
-        $mkmetalink $mirrors_list "$1/repodata/repomd.xml" \
+    "$createrepo" $OPTS --update -x 'fc*/*' -o "$1" -- "$1" >/dev/null || return
+    qubes-gpg-client-wrapper --detach-sign --armor -- "$1/repodata/repomd.xml" || return
```

Comment:
A couple things:
 - don't hardcode qubes-gpg-client-wrapper here. it should be configurable (GNUPG variable), and default to the same thing as for rpm packages - see how rpmsign_client variable is set in [builder-rpm](https://github.com/QubesOS/qubes-builder-rpm/blob/master/Makefile.rpmbuilder#L180)
 - key pointed by $SIGN_KEY option (and skip signing if not set)
 - name output explicitly, `qubes-gpg-client-wrapper` (contrary to `gpg2`) prints output to stdout by default